pkg/kubectl/cmd/util/openapi/validation/types.go

```diff
@@ -81,11 +63,11 @@ func (item *mapItem) sortedKeys() []string {
 var _ ValidationItem = &mapItem{}
 
 func (item *mapItem) VisitPrimitive(schema *openapi.Primitive) {
-	item.AddValidationError(InvalidTypeError{Path: schema.GetPath().String(), Expected: schema.Type, Actual: "map"})
+	item.AddValidationError(openapi.InvalidTypeError{Path: schema.GetPath().String(), Expected: schema.Type, Actual: "map"})
```

Comment:
`InvalidTypeError` is a validation error, I'm not sure why it's coming from `openapi`

Reply:
I'm reusing it in my code.